resources/migrations/20211213061358-remove-spaces-licence-plate.up.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE
+  cabs
+SET
+  licence_plate = regexp_replace(licence_plate, '[^a-zA-Z\d:]', '', 'g');
```

Comment:
Do we need to add a CONSTRAINT to the DB for this case?

Reply:
We are not changing any table schema, right?
Also, which CONSTRAINT are you thinking about?

Ps: this file is removed now, because we are running the data migration through the script.